src/components/Html.tsx

```diff
@@ -0,0 +1,23 @@
+import rehypePrism from "@mapbox/rehype-prism";
```

Comment:
should we adopt single quotes and no semicolons so code looks a bit alike across transloadit?

Reply:
done 👍